### ISSUE TITLE
Chart error since the addition of deepmerge in build 2.8.4

### ISSUE THREAD

**Issue by woodjamie** [apertureless/vue-chartjs#192]
### Actual Behavior
Since build 2.8.4 our charts no longer display any data. 

Console displays `[Vue warn]: Error in mounted hook: "TypeError: undefined is not an object (evaluating 'Object.keys(source)')"`

When following the trace the source arguement is undefinied in the function below.
```js
function deepmerge(target, source, optionsArgument) {
...
}
```
Pinning the version back to 2.8.3 resolves the issue.

### Environment
- vue.js version: 2.4.2
- vue-chart.js version: 2.8.4
- npm version: 3.10.10

**Comment by apertureless** [comment]
Can you show the options you are passing in?

**Comment by woodjamie** [comment]
## My component 
```js
<script>
import { Line, mixins } from 'vue-chartjs'

export default Line.extend({
  mixins: [mixins.reactiveProp],
  props: ['chartData', 'options'],
  mounted () {
    this.renderChart(this.chartData, this.chartOptions);
  }
})
</script>
```
## Template
```html
<div class="tab-content">
    <linechart :width="800" :height="300" :chartData="chartData" :Options="chartOptions" />
</div>
```
## Data
```js
                chartData: {
                    datasets: []
                },
                chartOptions: {
                    scales: {
                        yAxes: [{

                        }]
                    }
                },
...
```
Where datasets is dynamic.

**Comment by apertureless** [comment]
Your `renderChart()` should look like this:

```js
  this.renderChart(this.chartData, this.options);
```

as your props are `chartData` and `options` .

**Comment by apertureless** [comment]
However it seems to be related to `deepmerge` . Will check on this.

**Comment by apertureless** [comment]
Fixed in `2.8.5`
